identity-apps-core/apps/recovery-portal/src/main/webapp/sms-otp.jsp

```diff
@@ -339,8 +339,14 @@
                                     String baseUrl = ServiceURLBuilder.create()
                                         .addPath("/accountrecoveryendpoint/recoveraccountrouter.do").build()
                                         .getRelativePublicURL();
-                                    String multiOptionPathWithQuery = urlQuery + "&username=" + username 
-                                        + "&selectedOption=SMSOTP";
+                                    String multiOptionPathWithQuery;
+                                    if (urlQuery.contains("&username=")) {
+                                        String regex = "(&username=)[^&]+";
```

Comment:
We don't need to define this if this is not reused, IMO.

If we decide to keep the variable, let's rename it for better readability.

Reply:
Removed the variable with https://github.com/wso2/identity-apps/pull/6696/commits/7eaef08ce4f443e5e1c33606aae224f9ff8f1046